src/VisualStudio/Core/Impl/ProjectSystem/CPS/CPSProject_IWorkspaceProjectContext.cs

```diff
@@ -120,6 +120,9 @@ public void SetOptions(string commandLineForOptions)
             ExecuteForegroundAction(() =>
             {
                 var commandLineArguments = SetArgumentsAndUpdateOptions(commandLineForOptions);
+
+                SetRuleSetFile(commandLineArguments.RuleSetPath);
```

Comment:
Could you clarify, what does rule set file have to do with EnC for C# 7 features?